data/phi1348/abo013/phi1348.abo013.perseus-lat2.xml

```diff
@@ -17,27 +17,46 @@
           <name>William Merrill</name>
           <name>David Smith</name>
         </respStmt>
-        <funder n="org:FIPSE">Fund for the Improvement of Postsecondary Education</funder> 
+        <funder n="org:FIPSE">Fund for the Improvement of Postsecondary Education</funder>
       </titleStmt>
       <publicationStmt>
         <publisher>Trustees of Tufts University</publisher>
         <pubPlace>Medford, MA</pubPlace>
         <authority>Perseus Project</authority>
         <date type="release">2000-08-01</date>
       </publicationStmt>
-      
+
 
       <sourceDesc>
-        <bibl><idno type="ISBN">3519018276</idno></bibl>
+        <biblStruct>
+          <monogr>
+            <title xml:lang="lat">De Vita Caesarum Libri VIII</title>
+            <author>Suetonius</author>
```

Comment:
I'm confused because the print uses the full name (which is above in this file). Usually we use the common name or catalog name as the author in the header and then reproduce the print edition name here in the monograph part. So I'm not sure how this batch is working but I'm wondering about that consistency.

Reply:
I might have gotten confused and done it the other way, I know I have Suetonius as the author name in all of the cts_.xml files but pretty sure I have the full name in the headers. I can easily switch that and have the short name in the header and cts and the full name in the structured bibliography.